src/gui/codeeditors/qgscodeeditorpython.cpp

```diff
@@ -256,9 +356,9 @@ const char *QgsQsciLexerPython::keywords( int set ) const
 {
   if ( set == 1 )
   {
-    return "True False and as assert break class continue def del elif else except exec "
+    return "True False and as assert break class continue def del elif else except "
            "finally for from global if import in is lambda None not or pass "
-           "print raise return try while with yield";
+           "raise return try while with yield async await nonlocal";
```

Comment:
why these changes?

Reply:
print and exec are no longer keywords for Python2. However, async, await and nonlocal were added as keywords in Python3

Reply:
Ok, makes sense 👍